backend/src/main/java/gov/cdc/usds/simplereport/db/repository/TestOrderRepository.java

```diff
@@ -26,7 +26,15 @@
   String RESULT_RECENT_ORDER = " order by q.updatedAt desc ";
 
   @Query(FACILITY_QUERY + IS_PENDING + ORDER_CREATION_ORDER)
-  @EntityGraph(attributePaths = {"patient"})
+  @EntityGraph(
+      attributePaths = {
+        "patient",
+        "organization",
+        "deviceType",
+        "specimenType",
+        "results",
+        "deviceSpecimen"
+      })
```

Comment:
In stead of `Lazy` loading this fields, this will switch it to `Eager` loading for these fields, thus removing the need for dataloaders.
chose all the fields that are loaded in this graphql query, `fetchQueue()` is only used to resolve the `queue` in graphql

https://github.com/CDCgov/prime-simplereport/blob/91c9ac562a9f9e6a5d114820ce22fac6705ce8f4/frontend/src/app/testQueue/TestQueue.tsx#L61-L132